python_modules/dagster/docs/reference.rst

```diff
@@ -0,0 +1,212 @@
+Reference
+---------
+
+
+Solid
+^^^^^
+
+A solid is a functional unit of computation. Solids define their (optionally typed) inputs
+and outputs and the typed schema by which they can be configured, and can enfore expectations on
+their outputs.
+
+Solids can be strung together into `pipelines <#pipeline>`__ by defining
+`dependencies <#dependency-definition>`__ between their inputs and outputs.  Solids are reusable
+and instances of a solid may appear many times in a given pipeline, or across many different
+pipelines.
+
+Solids often wrap code written in or intended to execute in other systems (e.g., SQL statements,
```

Comment:
It might be good to move this (or the spirit of it) in to the first paragraph. The thinking being that we could hit more on the core of what a solid is before talking about the "optional" input/output metadata.

Reply:
yep